common/data/licenses.js

```diff
@@ -57,4 +51,33 @@ export default {
       'Creative Commons Attribution Non-Commercial No-Derivatives (CC BY-NC-ND 4.0) terms and conditions <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">https://creativecommons.org/licenses/by-nc-nd/4.0/</a>',
     ],
   },
+  'CC-BY-ND': {
+    url: 'https://creativecommons.org/licenses/by-nd/4.0/',
+    text: 'CC BY-ND',
+    icons: ['cc', 'cc_by', 'cc_nd'],
+    description:
+      'Free to use with attribution. No modifications permitted.',
+  },
+  'CC-BY-SA': {
+    url: 'https://creativecommons.org/licenses/by-sa/4.0/',
+    text: 'CC BY-SA',
+    icons: ['cc', 'cc_by']
```

Comment:
I think I need `icons` and `description`s for these new licenses. What happens if they don't exist?